src/main/java/net/sf/jabref/importer/fetcher/CrossrefFetcherTest.java

```diff
@@ -0,0 +1,83 @@
+package net.sf.jabref.importer.fetcher;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.ParserResult;
+import net.sf.jabref.importer.fileformat.BibtexParser;
+import net.sf.jabref.logic.util.DOI;
+import net.sf.jabref.model.database.BibDatabase;
+import net.sf.jabref.model.entry.BibEntry;
+
+public class CrossrefFetcherTest {
```

Comment:
I don't know :wink: If it will be useful for further improvements leave it - if it has already done its part remove it.